openapi/data_repository_service.swagger.yaml

```diff
@@ -396,3 +405,15 @@ definitions:
       license:
         type: object
         description: License information for the exposed API
+  BundleObjects:
+    type: object
+    properties:
+      name:
+        type: string
```

Comment:
please add a description (since it's not obvious that this string represents the name that should be used when materializing the object, and that the name here overrides any name that might be associated with the object itself)

Reply:
also -- should we be explicit that this string is not allowed to contain slashes?

Reply:
Description added.